### PR TITLE
Week 2 - coded CSS styles

### PR DESCRIPTION
**Pull request by tokyopdm**
Added new CSS for hovering over list items on hobbies page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    list-style-type: none;
+    border-radius: 12px;
+    width: 30%;
+    height: 100px;
+    background: dodgerblue;
+    text-align: center;
+    line-height: 100px;
+    transition: .5s;
+}
+.list-tiles li:nth-child(2) {
```
Comment (tokyopdm) [inline style.css:31]: This is a comment

**Review by tokyopdm — COMMENTED** [review]
Reviewed my own code lol

**Comment by tokyopdm** [comment]
This is a comment

**Review by liannabagarino — COMMENTED** [review]
CSS looks great to me!

**Pull request merged**